charts/pelorus/templates/tests/pelorus-prometheus-test-job.yml

```diff
@@ -0,0 +1,68 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: pelorus-prometheus-test
+  annotations:
+    "helm.sh/hook": test
+    "helm.sh/hook-delete-policy": before-hook-creation
+spec:
+  parallelism: 1
+  completions: 1
+  template:
+    metadata:
+      name: pelorus-prometheus-test
+    spec:
+      containers:
+      - name: prometheus-targets-test
+        image: docker.io/dwdraju/alpine-curl-jq # TODO: decide  what image to use...
```

Comment:
This might not work in some client environments.

Reply:
Please see my previous comment https://github.com/redhat-cop/pelorus/pull/227#issuecomment-721002502

Reply:
Could you use a pattern similar to this?  https://github.com/etsauer/pelorus/blob/grafana-snapshots/charts/pelorus/templates/dashboard-snapshot-cronjob.yaml#L58

Reply:
Yes, I will do. On the other hand, that approach won't work on disconnected environments.

Reply:
I've created issue #237 to address the image issue.  At least now we're using the same pattern across all the jobs.